ipaplatform/fedora/services.py

```diff
@@ -24,14 +24,22 @@
 
 from __future__ import absolute_import
 
+import os
+
 from ipaplatform.redhat import services as redhat_services
 
 # Mappings from service names as FreeIPA code references to these services
 # to their actual systemd service names
 fedora_system_units = redhat_services.redhat_system_units.copy()
 
-# Service that sets domainname on Fedora is called fedora-domainname.service
-fedora_system_units['domainname'] = 'fedora-domainname.service'
+# Fedora 28 and earlier have fedora-domainname.service. Starting from
+# Fedora 29, the service is called nis-domainname.service like on RHEL.
```

Comment:
The service is named rhel-domainname.service in RHEL. Just the comment is wrong.